From af408269cfbe58a556dba90b61afe8eba5315acf Mon Sep 17 00:00:00 2001
From: future <commanda.keen@gmail.com>
Date: Thu, 12 Dec 2013 12:31:57 +0100
Subject: [PATCH] Make caskroom configurable via command line argument

---
 rubylib/cask/cli.rb | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/rubylib/cask/cli.rb b/rubylib/cask/cli.rb
index 7778d3d..ad18977 100644
--- a/rubylib/cask/cli.rb
+++ b/rubylib/cask/cli.rb
@@ -59,6 +59,9 @@ class Cask::CLI
 
   def self.parser
     @parser ||= OptionParser.new do |opts|
+      opts.on("--caskroom=MANDATORY") do |v|
+        Cask.caskroom = Pathname(v).expand_path
+      end
       opts.on("--appdir=MANDATORY") do |v|
         Cask.appdir = Pathname(v).expand_path
       end
-- 
1.8.1

